services/movie_session.py

```diff
@@ -42,3 +42,10 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
```

Comment:
Consider using `List[Dict[str, int]]` as the return type hint for `get_taken_seats` to provide more clarity on the structure of the returned list.